### PR TITLE
LIFX : Add support for RefreshType.REFRESH

### PR DESCRIPTION
**Pull request by kgoderis**
Signed-off-by: Karel Goderis <karel.goderis@me.com>

### CODE REVIEW

File: extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/handler/LifxLightHandler.java [hunk 1]
```diff
@@ -230,34 +230,59 @@ public void initialize() {
...
+                        sendPacket(colorPacket);
+                        break;
+                    case CHANNEL_TEMPERATURE:
+                        sendPacket(colorPacket);
+                        break;
+                    default:
+                        break;
+                }
+            } catch (Exception ex) {
+                logger.error("Error while refreshing a chanel for the bulb: {}", ex.getMessage(), ex);
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/handler/LifxLightHandler.java]: typo: chanel

File: extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/handler/LifxLightHandler.java [hunk 2]
```diff
@@ -230,34 +230,59 @@ public void initialize() {
...
+            }
+
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.lifx/src/main/java/org/eclipse/smarthome/binding/lifx/handler/LifxLightHandler.java]: could you remove the 2 empty lines around the else?

**Comment by kgoderis** [comment]
@kaikreuzer done.

**Comment by kaikreuzer** [comment]
Thanks!

**Pull request merged**